src/message.c

```diff
@@ -22,6 +22,8 @@ static void msg_home_replace_attr(char_u *fname, int attr);
 static void msg_puts_attr_len(char *str, int maxlen, int attr);
 static void msg_puts_display(char_u *str, int maxlen, int attr, int recurse);
 static void msg_scroll_up(void);
+static void mch_msg_len(char *, int);
```

Comment:
Need `#ifdef USE_MCH_ERRMSG ~ #endif` to avoid macro (`mch_msg_len()` in vim.h) expansion.